mitodl/open-discussions
# Implemented authentication

#### What are the relevant tickets?
Fixes #44 

#### What's this PR do?
Uses the `refresh_token` plugin to authenticate with reddit, and 

#### How should this be manually tested?
There's nothing in this PR, but you should be able to see channels subscribed to in the REST API PR for only the user you are logged in as.
